automation/src/test/scala/org/broadinstitute/dsde/test/api/SnapshotAPISpec.scala

```diff
@@ -193,7 +198,7 @@ class SnapshotAPISpec extends AnyFreeSpecLike with Matchers with BeforeAndAfterA
 
           // check that the bq dataset has been created
           val workspaceId = resources.head.getMetadata.getWorkspaceId
-          val dataset = getDataset(generateReferenceName(workspaceId), projectName, ownerAuthToken)
+          val dataset = getDataset(generateReferenceName(workspaceId), googleProject.value, ownerAuthToken)
```

Comment:
when does `googleProject.value` differ from `projectName`?

Reply:
@sagehen03 this is the main thrust of project-per-workspace: the separation of billing projects and Google projects. The `projectName` only refers to billing, while `googleProject` refers to the Google project associated with the workspace. Any new workspace created after PPW is deployed (including the workspaces created in these tests) will have different values for these. (one could argue for renaming `projectName` inside this code!)

Reply:
`projectName` here represents the _Terra billing project_ (aka the workspace's namespace), while `googleProject.value` represents the _Google project_ associated with the workspace.

Reply:
Gotcha! ty

Reply:
I'd be all for disambiguating here and changing `projectName` to `wsNameSpace` or some such if it's not too much trouble.  A context dependent `project` meaning seems likely to cause problems for developers new to DSP.

Reply:
(Oh I commented, then refreshed and saw David's response :) )

Reply:
Yeah, good call to rename `projectName`

Reply:
Nit: I renamed `projectName` to `billingProject` for the entire test class. However, what do you all think about renaming to `workspaceNamespace` (which is what we use in swagger, etc) instead? I think it's too close in resemblance to `workspaceName`, but I don't feel too strongly about this. It's an easy refactor if further change is needed.